src/MS-RL.xml

```diff
@@ -0,0 +1,95 @@
+<spdx name="Microsoft Reciprocal License" identifier="MS-RL" osi-approved="true">
+  <urls>
+    <url>http://www.microsoft.com/opensource/licenses.mspx</url>
+    <url>http://www.opensource.org/licenses/MS-RL</url>
+  </urls>
+  <license>
+    <title>
+      <p>Microsoft Reciprocal License (Ms-RL)</p>
+    </title>
+    <body>
+      <p>This license governs use of the accompanying software. If you use the software, you accept this license.
+         If you do not accept the license, do not use the software.</p>
+      <list>
+        <li>
+          <b>1.</b>
+          <p>Definitions 
+            <br/>The terms &quot;reproduce,&quot; &quot;reproduction,&quot; &quot;derivative works,&quot;
+                 and &quot;distribution&quot; have the same meaning here as under U.S. copyright law. 
+          </p>
+          <p>A &quot;contribution&quot; is the original software, or any additions or changes to the software.</p>
+          <p>A &quot;contributor&quot; is any person that distributes its contribution under this license.</p>
+          <p>&quot;Licensed patents&quot; are a contributor&apos;s patent claims that read directly on its
+             contribution.</p>
+        </li>
+        <li>
+          <b>2.</b>
+          <p>Grant of Rights</p>
+        </li>
+        <list>
```

Comment:
Need to move sub-lists within the outer li tag, otherwise they do not appear in xml editor.